tests/EdFi.Ods.Api.IntegrationTests/StaffUSIToEducationOrganizationId.cs

```diff
@@ -0,0 +1,203 @@
+        {
+            var staffUniqueId = Guid.NewGuid().ToString("N");
+
+            Builder.AddLocalEducationAgency(1234)
+                   .AddSchool(4321, 1234)
+                   .AddStaff(staffUniqueId)
+                   .Execute();
+
+            Builder.StaffEducationOrganizationEmploymentAssociation(4321, staffUniqueId)
+                   .Execute();
+
+
+            AuthorizationViewHelper.ShouldContainStaffInSchoolOrDistrict(Connection, "StaffUSIToEducationOrganizationId", staffUniqueId, new List<int>() { 4321, 1234 });
+        }
+
+        [Test]
+        public void When_staff_is_assigned_to_a_school_thats_part_of_a_district_district_and_school_should_have_access_to_staff()
+        {
+            var staffUniqueId = Guid.NewGuid().ToString("N");
+
+            Builder.AddLocalEducationAgency(1234)
+                   .AddSchool(4321, 1234)
+                   .AddStaff(staffUniqueId)
+                   .Execute();
+
+            Builder.StaffEducationOrganizationAssignmentAssociation(4321, staffUniqueId)
+                   .Execute();
+
+            AuthorizationViewHelper.ShouldContainStaffInSchoolOrDistrict(Connection, "StaffUSIToEducationOrganizationId", staffUniqueId, new List<int>() { 4321, 1234 });
+        }
+
+        [Test]
+        public void When_staff_is_assigned_to_a_school_thats_part_of_a_district_no_other_district_or_school_should_have_access_to_staff()
+        {
+            var staffUniqueId = Guid.NewGuid().ToString("N");
+
+            Builder.AddLocalEducationAgency(1234)
+                   .AddSchool(4321, 1234)
+                   .AddStaff(staffUniqueId)
+                   .Execute();
+
+            Builder.StaffEducationOrganizationAssignmentAssociation(4321, staffUniqueId)
+                   .Execute();
+
+            AuthorizationViewHelper.ShouldNotContainStaffInOtherSchoolOrDistrict(Connection, "StaffUSIToEducationOrganizationId", staffUniqueId, new List<int>() { 4321, 1234 });
+        }
+
+        [Test]
+        public void When_staff_is_employed_to_a_school_thats_part_of_a_district_no_other_district_or_school_should_have_access_to_staff()
+        {
+            var staffUniqueId = Guid.NewGuid().ToString("N");
+
+            Builder.AddLocalEducationAgency(1234)
+                   .AddSchool(4321, 1234)
+                   .AddStaff(staffUniqueId)
+                   .Execute();
+
+            Builder.StaffEducationOrganizationEmploymentAssociation(4321, staffUniqueId)
+                   .Execute();
+
+            AuthorizationViewHelper.ShouldNotContainStaffInOtherSchoolOrDistrict(Connection, "StaffUSIToEducationOrganizationId", staffUniqueId, new List<int>() { 4321, 1234 });
+        }
+
+        [Test]
+        public void Should_not_contain_duplicates_in_view()
+        {
+            AuthorizationViewHelper.ShouldNotContainDuplicatesInView(Connection, "StaffUSIToEducationOrganizationId");
```

Comment:
Tests should try not to rely on Populated Template data, particularly for specific tests like this. With minimal control over what data is in Grand Bend, its unclear what this test is trying to prove. We should be setting up data that would raise concerns about generating duplicates (such as in this case, a staff who is employed/assigned to varying combinations of school/distrct) and make sure we have no duplicates in those scenarios.